src/main/java/com/movie/store/Main.java

```diff
@@ -63,8 +63,16 @@ public static void main(String[] args) throws AuthenticationException {
                 (UserService) injector.getInstance(UserService.class);
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
-        log.info("Register new user " + authenticationService.register("best@email.ever", "1234"));
-        log.info("Find user by email " + authenticationService.login("best@email.ever", "1234"));
+        try {
+            authenticationService.register("best@email.ever", "1234");
+        } catch (Exception e) {
+            log.error("Can't register new user. ", e);
+        }
+        try {
+            authenticationService.login("best@email.ever", "1234");
+        } catch (Exception e) {
+            log.error("Log in error. ", e);
```

Comment:
```suggestion
            log.warn("Log in error. ", e);
```
nothing really bad happened, `warn` level is enough

also, I can see one Sout left in this class below